src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs

```diff
@@ -138,6 +140,8 @@ internal sealed class VisualStudioProject
             _dynamicFileInfoProviders = dynamicFileInfoProviders;
             _hostDiagnosticUpdateSource = hostDiagnosticUpdateSource;
 
+            _telemetryService = _workspace.Services.GetRequiredService<IWorkspaceTelemetryService>();
```

Comment:
📝 I don't see a default implementation of this interface, so it seems possible for it to be missing in some scenarios. We should verify a default or make sure to handle cases where the service is missing.